charts/opentelemetry-kube-stack/charts/crds/README.md

```diff
@@ -14,5 +14,7 @@ Right now, upgrades are NOT handled by this chart, however that could change in
 ```bash
 wget https://raw.githubusercontent.com/open-telemetry/opentelemetry-operator/main/config/crd/bases/opentelemetry.io_opentelemetrycollectors.yaml
 wget https://raw.githubusercontent.com/open-telemetry/opentelemetry-operator/main/config/crd/bases/opentelemetry.io_opampbridges.yaml
-wget https://raw.githubusercontent.com/open-telemetry/opentelemetry-operator/main/config/crd/bases/opentelemetry.io_instrumentations.yaml\n
+wget https://raw.githubusercontent.com/open-telemetry/opentelemetry-operator/main/config/crd/bases/opentelemetry.io_instrumentations.yaml
+wget https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.75.0/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
```

Comment:
Is tagging a specific version here required? If yes we need to document a upgrade process

Reply:
we should be keeping with the compatibility of the operator and prometheus operator, i will link that here.